Formula/servus.rb

```diff
@@ -0,0 +1,110 @@
+class Servus < Formula
+  desc "Library and Utilities for zeroconf networking"
+  homepage "https://github.com/HBPVIS/Servus"
+  url "https://github.com/HBPVIS/Servus/archive/1.5.0.tar.gz"
+  sha256 "42fb9c060f17f040ad3c7563f5e87c89f5a221a5aa7da21384fc26b9c725ecc8"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    # Embed "serializeable" test from the servus 1.5.0 source
+    (testpath/"test.cpp").write <<-EOS.undent
+#define BOOST_TEST_MODULE servus_serializable
```

Comment:
Indent all the lines by six spaces. Ideally strip this test down so it's a bit more minimal i.e. just checks a link works but feel free to leave it as-is.